savvy-bindgen/src/gen/templates/configure

```diff
@@ -32,17 +32,23 @@ if [ "$(uname)" = "Emscripten" ]; then
   TARGET="wasm32-unknown-emscripten"
 fi
 
-# allow overriding profile on CI
-if [ -n "${PROFILE_OVERRIDE}" ]; then
-  PROFILE="${PROFILE_OVERRIDE}"
+# allow overriding profile externally (e.g. on CI)
+if [ -n "${SAVVY_PROFILE}" ]; then
+  PROFILE="${SAVVY_PROFILE}"
 # catch DEBUG envvar, which is passed from pkgbuild::compile_dll()
 elif [ "${DEBUG}" = "true" ]; then
   PROFILE=dev
 else
   PROFILE=release
 fi
 
+# e.g. SAVVY_FEATURES="a,b"  -->  "--features a --features b"
+if [ -n "${SAVVY_FEATURES}" ]; then
+  FEATURE_FLAGS="--features $(echo -n ${SAVVY_FEATURES} | sed 's/,/ --features /g')"
```

Comment:
IIUC, `--features ${SAVVY_FEATURES}` works this case.
https://doc.rust-lang.org/cargo/reference/features.html#command-line-feature-options